frontend/src/app/page.tsx

```diff
@@ -0,0 +1,31 @@
+"use client"
```

Comment:
Shouldn't this go into `src/pages`, according to [documentation](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts)?

Reply:
It depends on the approach used for the navigation. In this case, we are using the other option for the navigation named "App router", you can see the options at the top of the NextJS docs.
But we need to use the pages router option to improve the experience for the Authentication session as I said in other conversations. So I will define routing with the pages router approach.
<img width="321" alt="image" src="https://github.com/user-attachments/assets/83f3a813-87fc-4164-8edd-1c228a534e88">

Reply:
It depends on the approach used for the navigation. In this case, we are using the other option for the navigation named "App router", you can see the options at the top of the NextJS docs.
But we need to use the pages router option to improve the experience for the Authentication session as I said in other conversations. So I will define routing with the pages router approach.
<img width="385" alt="image" src="https://github.com/user-attachments/assets/d85bea69-0a1c-4912-af96-54b3a508620c">